kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionImpl.scala

```diff
@@ -118,6 +120,8 @@ class KyuubiSessionImpl(
     // we should call super.open before running launch engine operation
     super.open()
 
+    Option(tempFileService).foreach(_.addPathToExpiration(operationalLogRootDir.orNull))
```

Comment:
`addPathToExpiration` requires arg not null

Reply:
Yes. Let me add the guard check for it. BTW, the operationalLogRootDir is never null here as it's inited in L121 `super.open()`.